README.md

```diff
@@ -1,4 +1,4 @@
-# node-portfinder [![Build Status](https://api.travis-ci.org/http-party/node-portfinder.svg)](https://travis-ci.org/http-party/node-portfinder)
+# node-portfinder [![CI](https://github.com/http-party/node-portfinder/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/http-party/node-portfinder/actions/workflows/ci.yml)
```

Comment:
For GH actions, the badge text will be whatever the value of the `name` field in the yml file, unlike travis which used the title attribute from the markdown. I chose "CI" as my own convention for both here and within the CI.